src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -1124,6 +1125,258 @@ def test_filter_by_publisher_and_information_category(self):
                 },
             )
 
+    def test_facets(self):
```

Comment:
I'd rather see extra assertions added to the existing tests - we now have the tests like `test_filter_by_publisher_and_information_category` that also assert the returned aggregation results. This new test introduces another way to make the aggregation assertions, and that's confusing and inconsistent.